pep/app/constants/themes.ts

```diff
@@ -7,18 +7,54 @@ export interface Theme {
     id: ThemeId;
     cssPath: string;
     label: string;
+    colors: {
+        links: {
+            glossary: string;
+            bibliography: string;
+            pageReference: string;
+            external: string;
+            figure: string;
+            webExternal: string;
+            webInternal: string;
+            special: string;
+        };
```

Comment:
unless we have a good reason to do these in-JS, i think we should try to keep these defined directly in the SASS theme files so its part of the normal compiled CSS, like the rest of the theming is done currently. that gives us better integration w/existing bootstrap styles and reduces complexity/additional deps.

if the concern was making these values editable via admin configs, I had some discussions w/Alan about that early on, and would make the case that manual theme editing/control like that at a granular per element/color level might not be the best path (unintended design problems, accessibility issues, etc), as opposed to just having a the themes library managed and adjusted by devs in the codebase itself.

Reply:
Yes, these values per the spec are suppose to be editable from the admin. We can always adjust accordingly however for now, I think this should stay as is